test/e2e/shared/openstack.go

```diff
@@ -130,6 +130,8 @@ func DumpOpenStackPorts(e2eCtx *E2EContext, filter ports.ListOpts) (*[]ports.Por
 		_, _ = fmt.Fprintf(GinkgoWriter, "error creating provider client: %s\n", err)
 		return nil, err
 	}
+	// ToDo: remove comment
+	_, _ = fmt.Fprintln(GinkgoWriter, "custom port list=>%v<=", filter)
```

Comment:
this is a debug message? on purpose?

Reply:
Thanks , It will be removed. was debugging test failure.